### ISSUE TITLE
Anime OST patch

### ISSUE THREAD

**Issue by enumag** [07th-mod/higurashi-patch-compiler#22]
cc @Nozoki17

**Comment by Nozoki17** [comment]
I'll upload the patches (up to tata for now, I'm doing per-game patches to keep the size down, and so that I'm able to swap out music between question arcs and answer arcs for a greater variety) and do a nice text explaining everything tomorrow xD

**Comment by enumag** [comment]
@Nozoki17 Looking forward to it! :-)

**Comment by enumag** [comment]
Just found your previous [thread](http://steamcommunity.com/sharedfiles/filedetails/?id=956056614) about this. Although it only contains Oni and Wata patches.

**Comment by Nozoki17** [comment]
Yeah sorry, I have a shitton of stuff happening in my life right now and I just can't get myself to work for now.. I'm really sorry

**Comment by enumag** [comment]
Understood. Thanks for letting us know.
